test/nodejs-bundle/circuit/dial-over-any-relay.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-env mocha */
+'use strict'
+
+const TCP = require('libp2p-tcp')
+const WS = require('libp2p-websockets')
+const multiplex = require('libp2p-multiplex')
+
+const waterfall = require('async/waterfall')
+const utils = require('./helpers/utils')
+
+const chai = require('chai')
+chai.use(require('dirty-chai'))
+
+const expect = chai.expect
+
+describe('test connecting over any relay', function () {
+  this.timeout(500000)
```

Comment:
Without it, I would get intermittent timeouts without it, specially with WS which seem to take a while for the connection to get established... Definitely something to investigate once we iron everything else out.